scripts/prove_rpc.sh

```diff
@@ -108,49 +93,76 @@ fi
 # other non-proving code.
 if [[ $8 == "test_only" ]]; then
     # test only run
-    echo "Proving blocks ${BLOCK_INTERVAL} in a test_only mode now... (Total: ${TOT_BLOCKS})"
-    command='cargo r --release --package zero --bin leader -- --test-only --runtime in-memory --load-strategy on-demand --proof-output-dir $PROOF_OUTPUT_DIR --block-batch-size $BLOCK_BATCH_SIZE rpc --rpc-type "$NODE_RPC_TYPE" --rpc-url "$NODE_RPC_URL" --block-interval $BLOCK_INTERVAL  $PREV_PROOF_EXTRA_ARG --backoff "$BACKOFF" --max-retries "$RETRIES" '
+    echo "Proving blocks from ($START_BLOCK) to ($END_BLOCK)"
+    command="cargo r --release --package zero --bin leader -- \
+--test-only \
+--runtime in-memory \
+--load-strategy on-demand \
+--proof-output-dir $PROOF_OUTPUT_DIR \
+--block-batch-size $BLOCK_BATCH_SIZE \
+rpc \
+--rpc-type $NODE_RPC_TYPE \
+--rpc-url $NODE_RPC_URL \
+--start-block $START_BLOCK \
+--end-block $END_BLOCK \
+--backoff $BACKOFF \
+--max-retries $RETRIES \
+$PREV_PROOF_EXTRA_ARG"
+
     if [ "$OUTPUT_TO_TERMINAL" = true ]; then
-        eval $command
+        eval "$command"
```

Comment:
Why double " for eval argument when `command` is already a quoted string?

Reply:
I use shellcheck in vscode and it will complain about any variable without double quotes unless its inside a `[[ ... ]]` block iirc.

https://www.shellcheck.net/wiki/SC2086

The word-splitting would be fine in this case. But the globbing would not be desired. We don't have any instances of `*` in the string so that point is also moot here, but doesn't hurt to follow the shellcheck advice either.

LMK if I'm missing a reason we should not double quote there.